src/RedCode/Currency/Rate/XML/XMLParser.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace RedCode\Currency\Rate\XML;
+
+class XMLParser
+{
+    /**
+     * @param $string
+     * @return \SimpleXMLElement
+     */
+    public function parse($string)
+    {
+        return new \SimpleXMLElement($string);
```

Comment:
What is the purpose to move it to external class?
You can reach the same effect just having SOAPLoader and mocking of it.